gulpfile.babel.js

```diff
@@ -61,9 +74,9 @@ function sass() {
 
 // Copy and compress images
 function images() {
-  return gulp.src('src/assets/img/*')
+  return gulp.src('src/assets/images/*')
```

Comment:
Yeah I checked with @rafibomb and we're not really versioning the template at this time, so PR against master is fine.  I'm going to try to carve out some time later today to actually test drive this, but on a readthrough it looked pretty good, so yeah lets go ahead and add readme updates as well.